client/extensions/woocommerce/state/reducer.js

```diff
@@ -7,7 +7,9 @@ import { combineReducers } from 'redux';
  * Internal dependencies
  */
 import ui from './ui/reducer';
+import settings from './settings/reducer';
 
 export default combineReducers( {
-	ui
+	ui,
```

Comment:
what is the difference between ui and settings. I like breaking settings into its own grouping. Largely because we will be hitting some of the same api endpoints